VehicleCode/TestingAndCalibration/ReceiveTesting/src/Car.cpp

```diff
@@ -94,19 +94,23 @@ void Car::initialize(FlexCAN_T4<CAN2, RX_SIZE_256, TX_SIZE_16> myCan, int saveDe
  * Checks for a shutdown command in case of an error
 */
 void Car::readSensors() {
-    // update go state and log state
-    updateState();
-
-    // keep the motor off if the car is not active
-    if (!goFast) {
-        if (millis() - lastOffSignal > repeatOffDelay) {
-            Serial.println("Car is not active.");
-            sendMotorSignal(1, 0, SHUTDOWN);
-            lastOffSignal = millis();
-        }
-    }
+    // keep the motor off if the car is not active (HANDLED BY THE FRONT DATA COLLECTOR)
```

Comment:
can you explain this change?

Reply:
Running off signal is now being handled by the DataCollector